src/main/resources/entity-types/inventory/simple_holdings_records.json5

```diff
@@ -836,6 +836,25 @@
       visibleByDefault: false,
       valueGetter: ":sourceAlias.jsonb->>'notes'",
     },
+    {
+      name: 'tenant_id',
+      sourceAlias: 'inst',
+      essential: true,
+      dataType: {
+        dataType: 'stringType',
+      },
+      isIdColumn: true,
+      queryable: true,
+      visibleByDefault: true,
+      valueGetter: "'${tenant_id}'",
+      source: {
+        columnName: 'tenant_id',
+        entityTypeId: '30a5cfad-1868-4f46-86b9-a6ef67e2d9bf',
+        type: 'fqm',
+        name: 'tenant_id'
+      },
+      ecsOnly: false,
+    },
```

Comment:
Literally copy/paste from instances 🙂  All I changed is the `source.entityTypeId`